src/modules/sensors/module.yaml

```diff
@@ -393,12 +393,49 @@ parameters:
                 38: Roll 90°, Pitch 68°, Yaw 293°
                 39: Pitch 315°
                 40: Roll 90°, Pitch 315°
+                100: Custom Euler Angle
             min: -1
-            max: 40
+            max: 100
```

Comment:
This is probably enough spacing, but just to be safe maybe we should set this to a much larger magic number.

How about either UINT8_MAX (255) or even INT32_MAX? Currently the underlying parameter is actually an int32_t, but could be optimized in the future.

Reply:
I think we should eventually only use the "custom rotation", so this parameter will be completely removed